Server/VikingVault.Services/TransferRequestService.cs

```diff
@@ -23,12 +27,27 @@ public TransferRequest AddTransferRequest(TransferRequest transferRequestData)
             {
                 _dbContext.Add(transferRequestData);
                 _dbContext.SaveChanges();
+                AddRequestNotification(transferRequestData);
             }
             catch (Exception e)
             {
                 throw new DatabaseException();
             }
             return transferRequestData;
         }
+
+        private void AddRequestNotification(TransferRequest transferRequestData)
+        {
+            int? idReceiver = _userCardService.FindUserIdByCardNumber(transferRequestData.CardNumberReciever);
```

Comment:
You make this a nullable int but then you don't check for null values.... *judges intensifies*
`(int)null` will not work, just saying

Reply:
Improvise.Adapt.Overcome

Reply:
I checked it